src/Serilog.Sinks.MSSqlServer/Sinks/MSSqlServer/MSSqlServerSink.cs

```diff
@@ -166,90 +166,66 @@ DataTable CreateDataTable()
+                            ColumnName = _columnOptions.Level.ColumnName ?? StandardColumn.Level.ToString()
+                        });
+                        break;
+                    case StandardColumn.TimeStamp:
+                        eventsTable.Columns.Add(new DataColumn
+                        {
+                            DataType = typeof(DateTime),
+                            ColumnName = _columnOptions.TimeStamp.ColumnName ?? StandardColumn.TimeStamp.ToString(),
+                            AllowDBNull = false
+                        });
+                        break;
+                    case StandardColumn.LogEvent:
+                        eventsTable.Columns.Add(new DataColumn
+                        {
+                            DataType = typeof(string),
+                            ColumnName = _columnOptions.LogEvent.ColumnName ?? StandardColumn.LogEvent.ToString()
+                        });
+                        break;
+                    case StandardColumn.Message:
+                        eventsTable.Columns.Add(new DataColumn
+                        {
+                            DataType = typeof(string),
+                            MaxLength = -1,
+                            ColumnName = _columnOptions.Message.ColumnName ?? StandardColumn.Message.ToString()
+                        });
+                        break;
+                    case StandardColumn.MessageTemplate:
+                        eventsTable.Columns.Add(new DataColumn
+                        {
+                            DataType = typeof(string),
+                            MaxLength = -1,
+                            ColumnName = _columnOptions.MessageTemplate.ColumnName ?? StandardColumn.MessageTemplate.ToString()
+                        });
+                        break;
+                    case StandardColumn.Exception:
+                        eventsTable.Columns.Add(new DataColumn
+                        {
+                            DataType = typeof(string),
+                            MaxLength = -1,
+                            ColumnName = _columnOptions.Exception.ColumnName ?? StandardColumn.Exception.ToString()
+                        });
+                        break;
+                    case StandardColumn.Properties:
+                        eventsTable.Columns.Add(new DataColumn
+                        {
+                            DataType = typeof(string),
+                            MaxLength = -1,
+                            ColumnName = _columnOptions.Properties.ColumnName ?? StandardColumn.Properties.ToString()
+                        });
+                        break;
                 }
-
-                eventsTable.Columns.Add(level);
-            }
-
-            if (_columnOptions.Store.Contains(StandardColumn.TimeStamp))
-            {
-                var timestamp = new DataColumn
-                {
-                    DataType = Type.GetType("System.DateTime"),
```

Comment:
Changed this line to `typeof(DateTime)`. I couldn't see any reason why it needed to be `Type.GetType("System.DateTime")` so just wanted to be consistent with the rest of the code.